### DISCUSSION TITLE
How to load GGUF?

### DISCUSSION THREAD

**Discussion by vitorfdl** [LykosAI/StabilityMatrix#995] [Q&A] [answered]
I'm loving the Stability Matrix UI for generation, but I couldn't figure out how to load GGUFs.

I did notice the option to change model loader, but my GGUFs don't appear to be selected in the dropdown. Is it suppose for the GGUF to be in any specific folder (other than the StableDifussion folder) ?
![image](https://github.com/user-attachments/assets/c905cbc5-cc2a-4b2c-9ef7-dbdc44db63d5)

**Comment by mohnjiles** [comment, marked as answer]
Hi, sorry about that, the GGUF models will need to go in the `Unet` folder instead of the regular `StableDiffusion` folder. This applies for most Flux checkpoints as well. 

Also you may need the GGUF extension for ComfyUI; you can install it from the puzzle-piece 🧩 icon on the Packages page & then search for GGUF like this:
![image](https://github.com/user-attachments/assets/5e832975-8ea4-4f2c-870f-dac25c581cd8)

**Reply by yamenshahin** [reply]
Update: Unet folder is the shaerd folder DiffusionModels
Data\Models\DiffusionModels

**Comment by dwojewod** [comment]
<img width="1005" height="488" alt="ggufs" src="https://github.com/user-attachments/assets/25c7e680-1176-47db-95e2-fdd5d3431e19" />


I have following files in my \Data\Models\DiffusionModels

Name
----
flux-schnell-dev-merge-q5-1.gguf
flux-schnell-dev-merge-q5-1.jpeg
flux-schnell-dev-merge-q5-1.json
flux1-schnell-Q5_K_S.gguf
flux1-schnell-Q5_K_S.jpeg
flux1-schnell-Q5_K_S.json


StabilityMatrix Checkpoint Manager sees both models, but is not able to load their previews and json metadata. Is this expected ?
